Formula/pike.rb

```diff
@@ -22,7 +21,6 @@ class Pike < Formula
   option "with-pdf", "Include PDF support"
   option "with-gl", "Include GL support"
   option "with-all", "Include all features"
-  option "with-machine-code", "Enables machine code"
```

Comment:
actually, --with-machine-code is the default. It only was required to be turned off because Apple shipped a version of GCC that couldn't compile it circa 10.7.